pkgs/tools/security/tessen/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenvNoCC
+, fetchFromSourcehut
+, pass
+, bemenu
+, passBackend ? pass
+, picker ? bemenu
+, wtype
+, wl-clipboard
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "tessen";
+  version = "2.1.2";
+
+  src = fetchFromSourcehut {
+    owner = "~ayushnix";
+    repo  = pname;
+    rev = "8758a09345f6eef24764de4a0efad737f12562c8";
+    sha256  = "sha256-U6obXpYzIprOJ+b3QiE+eDOq1s0DYiwM55qTga9/8TE=";
+  };
+
+  buildInputs = [ passBackend picker wtype wl-clipboard ];
```

Comment:
I assume these packages have to be available at runtime. In that case maybe it makes sense to move them to `propagatedUserEnvPkgs`?